judgments/bootstrap/judgment2.xsl

```diff
@@ -257,10 +257,12 @@
 </xsl:template>
 
 <xsl:template match="img">
-	<img>
-		<xsl:apply-templates select="@*" />
-		<xsl:apply-templates />
-	</img>
+	<xsl:if test="matches(@src, '\.gif|\.png|\.jpg|\.jpeg')">
```

Comment:
Should we include `webp` and `svg` for completeness while we're here? SVG might actually be the best thing to convert them to in the long run, as WMF can be vector or bitmap.

Reply:
```suggestion
	<xsl:if test="matches(@src, '\.gif|\.png|\.jpg|\.jpeg|\.webp|\.svg')">
```

Reply:
Good call